github-actions/verify-pr-title/package.json

```diff
@@ -13,7 +13,7 @@
 	},
 	"dependencies": {
 		"@actions/core": "^1.2.4",
-		"@actions/github": "^4.0.0"
+		"@actions/github": "^5.0.0"
```

Comment:
Same with #843, we had to increment the version number in order to bypass this issue: https://github.com/octokit/plugin-rest-endpoint-methods.js/issues/172